### ISSUE TITLE
Add option for persisting of notifications

### ISSUE THREAD

**Issue by notuxius** [plasma-extensions/plasma-widget-notification-sidebar#5]
Presently notifications are vanishing after timeout.

There can be a checkbox or a 'Persistent notifications' toggling button to the left side from 'Clear All' button inside widget or in Settings.

**Comment by koma111** [comment]
Yes, please! It is annoying that they disappear after timeout. A history-like list would be better, much better. With the notifications expanded ofc.
